SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/UserIdContext.cs

```diff
@@ -48,7 +57,27 @@ public static UserIdContext Instance
         /// <summary>
         /// Current user identifier.
         /// </summary>
-        public string UserId { get; set; }
+        public string UserId
+        {
+            get
+            {
+                lock (UserIdContextLock)
+                {
+                    return _userId;
+                }
+            }
+            set
+            {
+                lock (UserIdContextLock)
+                {
+                    if (!_userId.Equals(value))
```

Comment:
check other sdk does case sensitive or case-insensitive?

Reply:
Checked Android SDK is case sensitive.